src/index.js

```diff
@@ -33,17 +33,19 @@ function IntroJs(obj) {
 
   this._options = {
     /* Next button label in tooltip box */
-    nextLabel: "Next &rarr;",
+    nextLabel: "Next",
     /* Previous button label in tooltip box */
-    prevLabel: "&larr; Back",
+    prevLabel: "Back",
     /* Skip button label in tooltip box */
-    skipLabel: "Skip",
+    skipLabel: "×",
```

Comment:
Probably bad for accessibility. Suppose it may need to be provided for translations to though.

Reply:
correct